app/controllers/admin/works_controller.rb

```diff
@@ -113,6 +113,51 @@ def download_ohms_xml
       :disposition => ContentDisposition.format(disposition: "attachment", filename: "#{@work.oral_history_content!.ohms_xml.accession}.xml")
   end
 
+  # Biographical metadata form
+  # GET "/admin/works/ab2323ac/oh_bio_form"
+  def oh_biography_form
+    @work.oral_history_content!
+    @work.oral_history_content.
+      interviewee_birth ||= OralHistoryContent::DateAndPlace.new
+    @work.oral_history_content.
+      interviewee_death ||= OralHistoryContent::DateAndPlace.new
```

Comment:
I think you can avoid these `||=` lines by just making this a default in the model `attr_json`, so it defaults to a blank one.

Reply:
No, I need them as the code currently stands. I've written tests that cover the scenario.

Reply:
Can/should you replace em with attr_json defaults? In OralHistoryContent, 

     attr_json :interviewee_birth, OralHistoryContent::DateAndPlace, default: -> { OralHistoryContent::DateAndPlace.new }

Reply:
No, unfortunately that won't work. The default is only applied for new `OralHistoryContent`s, and if you delete the attribute, it needs to be recreated before showing the form-- otherwise the fields won't show up.

Reply:
OK, no problem, it's probably good enough at this point, but "if you delete the attribute" -- why would you ever delete the attribute?  Oh right, because you wrote code to delete it if you blank out all the fields!

Ok, it's good enough!